src/config/appSettings.ts

```diff
@@ -17,6 +17,9 @@ const appSettings = {
     (process.env.SHOULD_START_MONITORING_UI as unknown as string) === 'true',
   shouldProcessPastEvents:
     (process.env.SHOULD_PROCESS_PAST_EVENTS as unknown as string) === 'true',
+  cacheInvalidationEndpoint:
+    process.env.CACHE_INVALIDATION_ENDPOINT ||
+    shouldNeverHappen('CACHE_INVALIDATION_ENDPOINT is not set.'),
```

Comment:
Should be optional IMO and default to disabling the cache invalidation-related logic. You might not need / want this for all deployments of event-processor, especially considering caching is optional on the app side

Reply:
Good catch -- it's already treated as optional in other places but this is a leftover, removing.